app/views/layouts/application.html.slim

```diff
@@ -29,7 +27,7 @@ html
           ul.navbar-nav.ml-auto
             span.balance
               b
-                | Balance:
+                = t('balance') + ':'
```

Comment:
@marderer is 
```ruby
= t('balance')
| :
```
workable solution?

adding something to localized text via `+` looks like a hack 😄